salt/pillar/postgres.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+'''
+Retrieve Pillar data by doing a postgres query
+
+:maturity: new
+:depends: psycopg2
+:platform: all
+
+Complete example
+=====================================
+
+.. code-block:: yaml
+
+    postgres:
+      user: 'salt'
+      pass: 'super_secret_password'
+      db: 'salt_db'
```

Comment:
Sorry, I don't understand why the diff.  I'm not following the reasoning "in case something gets added in the future, we don't have to update the configuration options that salt accepts."  If something gets added, you don't have to add a config option for it?  Are you saying the master config would just define a URL instead of keys:values?